app/assets/stylesheets/modules/_navbar.scss

```diff
@@ -159,31 +161,24 @@
   }
 }
 
-.navbar-default.program-subnav {
+.navbar-fixed-top.program-subnav, .navbar-fixed-top.schedule-subnav {
   border-radius: 0;
   background-color: $dark-blue;
   border-color: $dark-blue;
-
-  li.active {
-    border-bottom: 3px solid darken($dark-blue, 10%);
-    > a , a:hover, a:focus {
-      color: darken($white, 10%);
-      background-color: lighten($dark-blue, 10%);
-    }
+  padding-top: $navbar-height;
+  z-index: $zindex-subnavbar;
+  li > a {
+    color: $white;
+  }
+  a:hover {
+    background-color: $dark-blue;
+    color: darken($white, 10%);
   }
-}
-
-.navbar-default.schedule-subnav {
```

Comment:
same styling as `.program-subnav`, so I deleted and added the `.schedule-subnav` selector to the former's styling